src/lib.rs

```diff
@@ -53,6 +53,8 @@ pub use types::{
 };
 use types::{Directory, JoseJson, Signer};
 
+const CRATE_USER_AGENT: &str = concat!("instant_acme/", env!("CARGO_PKG_VERSION"));
```

Comment:
Please move this down to somewhere near the bottom of the module (though before `mod tests {}`), to match the top-down style of the crate.

Also, please spell the crate name with a hyphen instead of an underscore.